.circleci/config.yml

```diff
@@ -171,7 +171,39 @@ jobs:
           no_output_timeout: 5h # Circl CI limits to 5hr max: https://discuss.circleci.com/t/job-times-out-after-5-hours/32220/3
           command: |
             . ~/venv/bin/activate
-            make runslow 
+            make runslow
+
+  Deploy to Rinkeby:
+    machine:
+      image: ubuntu-1604:201903-01
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run:
+          name: "Switch to Python v3.7"
+          command: |
+            pyenv versions
+            pyenv global 3.7.0
+      - run:
+          name: build docker
+          command: |
+            docker build -t omisego/plasma-contracts .
+      - run:
+          name: deploy contract to Rinkeby
+          no_output_timeout: 1h
+          command: |
+            docker run --rm \
+            -e DEPLOYER_PRIVATEKEY=${FAUCET_PRIV_KEY} \
+            -e MAINTAINER_PRIVATEKEY=${FAUCET_PRIV_KEY} \
+            -e AUTHORITY_PRIVATEKEY=${FAUCET_PRIV_KEY} \
+            -e REMOTE_URL="https://rinkeby.infura.io/v3/${INFURA_API_TOKEN}" \
+            omisego/plasma-contracts \
+            npx truffle migrate --reset \
+            --network remote
+      - run:
+          name: check faucet balance
+          command: |
+            python .circleci/scripts/check_faucet_balance_and_warn.py
```

Comment:
maybe let's move it up - even before building a docker image

Reply:
I put it here so that after using some eth from the faucet account, we can make sure the remaining is still enough.